src/components/FontAwesomeLayers.js

```diff
@@ -1,5 +1,8 @@
+import { computed, defineComponent, h } from 'vue'
 import { config } from '@fortawesome/fontawesome-svg-core'
-import { defineComponent, h, computed } from 'vue'
+import { ICON_PACKS_STARTING_VERSION, SVG_CORE_VERSION } from '../utils'
```

Comment:
I added this import here and the "semver" import.
The other imports were previously added, I just alphabetized them.